src/util/selinux.c

```diff
@@ -340,6 +341,17 @@ static int bus_selinux_log(int type, const char *fmt, ...) {
         return 0;
 }
 
+/**
+ * On a policy reload we need to reparse the SELinux configuration file, since
+ * this could have changed. The call back is registered in the broker, and
+ * the SIGHUP is caught in the launcher. So, send a SIGHUP to our parent to
+ * reload all configs.
+ */
+static int policy_reload_callback(int seqno) {
+        pid_t ppid = getppid();
+        return kill(ppid, SIGHUP);
```

Comment:
The launcher/controller is not necessarily the parent PID. We need to pass this through the controller interface, just like the other activation related messages.